packages/block-library/src/image/edit.js

```diff
@@ -246,10 +245,6 @@ class ImageEdit extends Component {
 		};
 	}
 
-	getImageSizes() {
-		return get( this.props.image, [ 'media_details', 'sizes' ], {} );
-	}
```

Comment:
Removing this function causes an exception as we're still calling `this.getImageSizes()` below on line 352.

```
react-dom.min.82e21c65.js:110 TypeError: this.getImageSizes is not a function
```